infra/config.ts

```
@@ -0,0 +1,65 @@
+    'middle': '│',
+  },
+  style: {
+    head: ['blue', 'bold'],
+    // compact: true,
+  },
+  // wordWrap: true,
+};
+
+
+export const emojiMap: Record<string, string> = {
+  email: "✉",
```

ESLint

[error] Strings must use singlequote.

(style/quotes)